lib/mixlib/install/generator/bourne.rb

```diff
@@ -46,7 +46,9 @@ def install_command
           install_command << get_script("helpers.sh")
           install_command << render_variables
           install_command << get_script("platform_detection.sh")
-          if options.for_artifactory?
+          # Since omnitruck can not resolve unstable we need to inject direct
+          # urls for the packages here.
+          if options.for_unstable?
```

Comment:
Perhaps we should just go ahead and enable the unstable channel in Omnitruck? It would make it easier to reason about things.